lib/motion-markdown-it-plugins/emoji/plugin/normalize_opts.rb

```diff
@@ -6,9 +6,16 @@ def quoteRE(str)
         str.gsub(/([.?*+^$\[\]\\(){}|-])/, '\\\\\1')
       end
 
+      # convert keys from symbols into strings
+      #------------------------------------------------------------------------------
+      def stringify(symbol_hash)
+        symbol_hash.collect{|k,v| [k.to_s, v]}.to_h
+      end
```

Comment:
just fyi - Rails ships with a [stringify_keys](https://apidock.com/rails/Hash/stringify_keys) method. And the core function was added to Ruby 2.5 [Ruby 2.5 adds Hash#transform_keys method](https://blog.bigbinary.com/2018/01/09/ruby-2-5-adds-hash-transform_keys-method.html)

Reply:
True, but since I'm supporting RubyMotion as well, I can't rely on Rails or Ruby 2.5. And I want to keep, for now, dependencies small.